assets/src/components/mapPage/routePropertiesCard.tsx

```diff
@@ -0,0 +1,259 @@
+                selectRoutePattern(targetRoutePattern)
+              }
+            }}
+          />
+          <label htmlFor={`direction-radio-${directionId}`}>
+            {directionNames[directionId as DirectionId]}
+          </label>
+        </div>
+      ))}
+    </div>
+  )
+}
+
+const VariantOption = ({
+  routePattern,
+  isSelected,
+  selectRoutePattern,
+}: {
+  routePattern: RoutePattern
+  isSelected: boolean
+  selectRoutePattern: (routePattern: RoutePattern) => void
+}) => {
+  return (
+    <div
+      className={`m-route-properties-card__variant-picker-variant ${
+        isSelected ? "selected" : ""
+      }`}
+    >
+      <input
+        type="radio"
+        id={`variant-radio-${routePattern.id}`}
+        name="variant"
+        defaultChecked={isSelected}
+        onChange={() => selectRoutePattern(routePattern)}
+      />
+
+      <label htmlFor={`variant-radio-${routePattern.id}`}>
+        <CircleCheckIcon className="variant-check" />
+        <div>
+          <div className="m-route-properties-card__variant-picker-name">
+            {displayName(routePattern)}
+          </div>
+          <div className="m-route-properties-card__variant-picker-time-description">
+            {routePattern.timeDescription && routePattern.timeDescription}
+          </div>
+        </div>
+      </label>
+    </div>
+  )
+}
+
+const VariantPicker = ({
+  routePatterns,
+  selectedRoutePatternId,
+  selectRoutePattern,
+}: {
+  routePatterns: RoutePattern[]
+  selectedRoutePatternId: RoutePatternId
+  selectRoutePattern: (routePattern: RoutePattern) => void
+}): JSX.Element => {
+  {
+    return (
+      <>
+        {sortRoutePatterns(routePatterns).map((routePattern) => (
+          <VariantOption
+            key={routePattern.id}
+            routePattern={routePattern}
+            isSelected={routePattern.id === selectedRoutePatternId}
+            selectRoutePattern={selectRoutePattern}
+          />
+        ))}
+      </>
+    )
+  }
+}
+
+const RoutePropertiesCard = ({
+  routePatterns,
+  selectedRoutePatternId,
+  selectRoutePattern,
+  onClose,
+}: {
+  routePatterns: ByRoutePatternId<RoutePattern>
+  selectedRoutePatternId: RoutePatternId
+  selectRoutePattern: (routePattern: RoutePattern) => void
+  onClose: () => void
+}) => {
+  const [openedDetails, setOpenedDetails] = useState<
+    "variants" | "stops" | null
+  >(null)
+
+  const selectedRoutePattern = routePatterns[selectedRoutePatternId]
+  const route: Route | null = useRoute(selectedRoutePattern?.routeId)
+
+  if (!route || selectedRoutePattern === undefined) {
+    return <></>
+  }
+  return (
+    <div className="m-route-properties-card" aria-label="route properties card">
+      <Card
+        style={"white"}
+        noFocusOrHover={true}
+        title={
+          <>
+            <div className="route-title">
+              <RoutePill routeName={routeId} />
+              <h4>{displayName(selectedRoutePattern)}</h4>
```

Comment:
question+suggestion: why `<h4>` here?

I think `<h2>` would be more approprate, but that's because I think of the RPC as is it's own section which restarts the header index to 2.